WindowsServerDocs/failover-clustering/deploy-cloud-witness.md

```diff
@@ -101,7 +101,7 @@ When you create a Storage Account, the following URLs are generated using the fo
 Cloud Witness always uses **Blob** as the storage type. Azure uses **.core.windows.net** as the Endpoint. When configuring Cloud Witness, it is possible that you configure it with a different endpoint as per your scenario (for example the Microsoft Azure datacenter in China has a different endpoint).
 
 > [!NOTE]
-> The endpoint URL is generated automatically by Cloud Witness resource and there is no extra step of configuration necessary for the URL.
+> The endpoint URL is generated automatically by Cloud Witness resource and there is no extra step of configuration necessary for the URL. You also need to ensure that the endpoint is whitelisted along with port 443 in order to connect to Azure Storage.
```

Comment:
```suggestion
> The endpoint URL is generated automatically by the Cloud Witness resource though you do need to make sure that port 443 is open in your firewalls and that ***.core.windows.net** is included in any firewall allow lists you're using between the cluster and Azure.
```

Reply:
@Amrinder-Singh29 - how are my suggested edits?

Reply:
Hi @JasonGerend - Yes, this looks good. 
Just one minor suggestion of adding a keyword "Storage" towards the last. i.e ".....using between the cluster and Azure Storage."